<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbImport">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textDataPath">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotOriginalData" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>611</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLoad">
    <property name="geometry">
     <rect>
      <x>637</x>
      <y>11</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Original Data</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>12</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbTimeStamp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>14</y>
      <width>99</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>With Timestamp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRun">
    <property name="geometry">
     <rect>
      <x>718</x>
      <y>11</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSave">
    <property name="geometry">
     <rect>
      <x>799</x>
      <y>11</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>228</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="sbWindowLength"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string> Data Window
 Length</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbWindowStep"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string> Data Window
 Step Size</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="tbResult">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>681</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotBadData" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bad Data</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbBadDataNum">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>pbRun</zorder>
   <zorder>pbSave</zorder>
   <zorder>pbImport</zorder>
   <zorder>textDataPath</zorder>
   <zorder>plotOriginalData</zorder>
   <zorder>cbTimeStamp</zorder>
   <zorder>pbLoad</zorder>
   <zorder>label</zorder>
   <zorder>progressBar</zorder>
   <zorder>tbResult</zorder>
   <zorder>plotBadData</zorder>
   <zorder>label_4</zorder>
   <zorder>sbBadDataNum</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionSave_to">
   <property name="text">
    <string>Save to</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actParamSetting">
   <property name="text">
    <string>Parameters</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
